Packs/IdentityThreatResponsePack/Playbooks/playbook-XSOAR_-_Silverfort_for_Incident_Response.yml

```diff
@@ -0,0 +1,1037 @@
+    id: "24"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "6"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      value:
+        simple: ${incident.details}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Prints text to war room (Markdown supported)
+      id: 98e4466b-0f60-4ffe-8e1f-9251597de0f5
+      iscommand: false
+      name: Retrieve alert details
+      script: Print
+      type: regular
+      version: -1
+    taskid: 98e4466b-0f60-4ffe-8e1f-9251597de0f5
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 510
+        }
+      }
+  "26":
+    id: "26"
+    ignoreworker: false
+    note: false
+    quietmode: 0
+    scriptarguments:
+      assetid: {}
+      closeNotes:
+        simple: All stakeholders have been updated
+      closeReason:
+        simple: Endpoint contained. Manager updated
+      code42alerttype: {}
+      emailclassification: {}
+      id:
+        complex:
+          accessor: id
+          root: incident
+      phishingsubtype: {}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Builtin
+      description: commands.local.cmd.close.inv
+      id: 97ca62de-2914-474a-875c-ef3599470351
+      iscommand: true
+      name: Close Investigation
+      script: Builtin|||closeInvestigation
+      type: regular
+      version: -1
+    taskid: 97ca62de-2914-474a-875c-ef3599470351
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 2205
+        }
+      }
+  "29":
+    continueonerror: true
+    evidencedata:
+      customfields: {}
+      description:
+        simple: alert the manager
+    id: "29"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "30"
+    note: true
+    quietmode: 0
+    scriptarguments:
+      additionalHeader: {}
+      attachCIDs: {}
+      attachIDs: {}
+      attachNames: {}
+      bcc: {}
+      body:
+        simple: |-
+          Data exfiltration activity has been detected by Code42 for user ${incident.labels.actor}. File exfiltration details:
+
+          Files:
+          ${incident.code42fileevents.filename}
+
+          Paths:
+          ${incident.code42fileevents.filepath}
+
+          Exposures:
+          ${incident.code42fileevents.exposure}
+
+          If you have access to Demisto, you can view the incident details at ${inputs.DemistoInstanceURL}#/incident/${incident.id}
+      bodyType: {}
+      cc: {}
+      flag: {}
+      from: {}
+      headers: {}
+      htmlBody: {}
+      importance: {}
+      raw_message: {}
+      replyTo: {}
+      subject:
+        simple: ${incident.name}
+      templateParams: {}
+      to:
+        simple: ${UserManagerEmail}
+      transientFile: {}
+      transientFileCID: {}
+      transientFileContent: {}
+      using:
+        simple: Mail Sender (New)_instance_1
```

Comment:
Please do not bind commands to specific instances (or integrations for that matter). For example, use "msgraph-direct-reports" and not "msgraph-direct-reports (Microsoft_Graph_instance1)".
Please fix it here and in other places in this file and in your other files.